test/Microsoft.Extensions.Configuration.UserSecrets.Test/ConfigurationExtensionTest.cs

```diff
@@ -43,6 +45,35 @@ private void SetSecret(string id, string key, string value)
         }
 
         [Fact]
+        public void AddUserSecrets_FindsAssemblyAttribute()
+        {
+            var configKey = nameof(AddUserSecrets_FindsAssemblyAttribute);
+            SetSecret("Microsoft.Extensions.Configuration.UserSecrets.Test", configKey, "true");
```

Comment:
Yeah I had to re-read this several times. The problem I had is that "true" and "false" have actual meanings. Changing these to be more meaningless, but perhaps also to be clearer about their semantic meaning, could certainly make this easier to understand.